site/Lightbox.tsx

```diff
@@ -42,69 +70,92 @@ export const Lightbox = ({
+    const onContentClick = useCallback(
+        (e: React.MouseEvent) => {
+            if (e.target === contentRef.current) {
+                onClose()
+            }
+        },
+        [onClose]
+    )
+
+    useTriggerOnEscape(onClose)
 
     return (
-        <div className="container">
-            {!isLoaded && (
-                <LoadingIndicator backgroundColor="#000" color="#ccc" />
-            )}
-            <TransformWrapper doubleClick={{ mode: "reset" }} smooth={false}>
-                {({ zoomIn, zoomOut, resetTransform }) => (
-                    <>
-                        <div
-                            className="content"
-                            ref={contentRef}
-                            onClick={(e) => {
-                                if (e.target === contentRef.current) {
-                                    close()
-                                }
-                            }}
-                        >
-                            <TransformComponent>
-                                {children(isLoaded, setIsLoaded)}
-                            </TransformComponent>
-                        </div>
-                        <div className="tools">
-                            {isLoaded && (
-                                <>
-                                    <button
-                                        aria-label="Zoom in"
-                                        onClick={() => zoomIn()}
-                                    >
-                                        <FontAwesomeIcon icon={faSearchPlus} />
-                                    </button>
-                                    <button
-                                        aria-label="Zoom out"
-                                        onClick={() => zoomOut()}
-                                    >
-                                        <FontAwesomeIcon icon={faSearchMinus} />
-                                    </button>
-                                    <button
-                                        aria-label="Reset zoom"
-                                        onClick={() => resetTransform()}
-                                    >
-                                        <FontAwesomeIcon icon={faCompress} />
-                                    </button>
-                                    <button
-                                        onClick={handleDownload}
-                                        aria-label="Download high resolution image"
-                                    >
-                                        <FontAwesomeIcon icon={faDownload} />
-                                    </button>
-                                </>
-                            )}
-                            <button
-                                aria-label="Close"
-                                onClick={close}
-                                className="close"
+        <BodyDiv divClassname="lightbox">
```

Comment:
`<BodyDiv>` renders the lightbox into a React Portal, i.e. a newly-created div that's appended to `document.body`.